pkg/api/util/usages.go

```diff
@@ -49,7 +49,7 @@ var extKeyUsages = map[cmapi.KeyUsage]x509.ExtKeyUsage{
 	cmapi.UsageTimestamping:    x509.ExtKeyUsageTimeStamping,
 	cmapi.UsageOCSPSigning:     x509.ExtKeyUsageOCSPSigning,
 	cmapi.UsageMicrosoftSGC:    x509.ExtKeyUsageMicrosoftServerGatedCrypto,
-	cmapi.UsageNetscapSGC:      x509.ExtKeyUsageNetscapeServerGatedCrypto,
+	cmapi.UsageNetscapeSGC:     x509.ExtKeyUsageNetscapeServerGatedCrypto,
```

Comment:
It's possible this is an API change

Reply:
It _is_ an API change from a Golang perspective, but the 'value' of the const has not changed so it's fine :)